src/wstool/wstool_cli.py

```diff
@@ -186,3 +186,15 @@ def wstool_main(argv=None, usage=None):
 
     except KeyboardInterrupt:
         return 1
+
+
+def main():
+    try:
+        from .common import MultiProjectException
+        sys.exit(wstool_main(sys.argv))
+    except ImportError as exc:
+        sys.exit("ERROR: Cannot find required rosinstall library version, \
+    check your installation (also of vcstools) is up-to-date. One frequent cause \
```

Comment:
I think this spacing is going to look really weird on the print-out, right?

Also, more pythonic ways of writing multiline strings are:
```
    sys.exit("Part of a string "
             "and the rest of it.")
```
Or
```
    sys.exit("""\
Part of a string \
and the rest of it.""")
```

Note the spacing.

Reply:
Fixed.